modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -282,6 +282,189 @@ enum Breakpoint {
   DISABLED
 }
 
+"""
+A single Call for Proposals definition.
+"""
+type CallForProposals {
+
+  """
+  The unique Call for Proposals id associated with this Call.
+  """
+  id: CallForProposalsId!
+
+  """
+  Open/closed status of this Call. Submissions are only accepted when the Call
+  is OPEN.
+  """
+  status: CallForProposalsStatus!
```

Comment:
I wonder if this should rather be dates start/end of the CfP

Reply:
Each partner has a deadline for submission, and there's a start and end time for when observing will happen associated with this Call.  This was just meant as a way of leaving it off until and whenever we aren't ready for them.